pkg/project/project.go

```diff
@@ -242,11 +247,8 @@ func (p *Project) Cache() string { return p.projectfile.Cache }
 
 // Namespace returns project namespace
 func (p *Project) Namespace() *Namespaced {
-	commitID, err := localcommit.Get(p.Dir())
-	if err != nil && !localcommit.IsFileDoesNotExistError(err) {
-		multilog.Error("Unable to get local commit: %v", errs.JoinMessage(err))
-	}
-	return &Namespaced{p.projectfile.Owner(), p.projectfile.Name(), &commitID, false}
+	commitId := strfmt.UUID(p.projectfile.LegacyCommitID())
+	return &Namespaced{p.projectfile.Owner(), p.projectfile.Name(), &commitId, false}
```

Comment:
Is the legacy commit ID still required as part of the namespace? Suggest dropping it altogether from the namespace.

Reply:
Having an eye towards https://activestatef.atlassian.net/jira/software/c/projects/DX/issues/DX-2289, I'd say we probably shouldn't drop it yet.

Reply:
Never mind. We abandoned during stand, so I removed it. Let's see what breaks.